migrations/sql/V1.10__Add_Documents.sql

```diff
@@ -1,10 +1,10 @@
 CREATE TABLE public.document_manager (
 	document_manager_id serial NOT NULL,
-	document_guid uuid NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
You can't change an exisiting migration. If you do need to remove/update these existing constraints. You should create a new migration.
Unless there was team concensus of deleting the test and prod environments and recreating them that i'm not aware of

Reply:
Right, I thought since this was the first time the migration was used that I could change it. I forgot it was a separate PR and already lived in the Sprint Branch I changed it to be done in a different script.